src/dbcp/data_mart/projects.py

```diff
@@ -595,8 +595,8 @@ def create_total_active_project_change_logs(
     ).all(), "Found rows with unexpected queue status."
 
     chng_log = active_iso_projects_change_log.copy()
-    min_date = chng_log.effective_date.min()
-    max_date = chng_log.effective_date.max()
+    min_date = chng_log.effective_date.min() - pd.offsets.QuarterBegin(startingMonth=1)
+    max_date = chng_log.effective_date.max() + pd.offsets.QuarterEnd(0)
```

Comment:
I discovered a bug where projects in the most recent quarter weren't being included in the change log table. This was happening because `pd.date_range(start="{date in quarter}", end="{maximum project date in the same quarter}", ...)` returns an empty series. I fixed this by setting the max date to the end day of the quarter.